modules/ProvBase/Resources/views/Configfile/edit.blade.php

```diff
@@ -73,6 +94,30 @@
         onKeyUp: function(newval, key, id) {
             this.lists[key].content[id].name=newval;
         },
+        itemmenu: function(element, key, id) {
+            targetElement=element.parentNode.getElementsByClassName("dragdropitemmenubox")[0];
+            if (targetElement.style.display!="block"){
+                targetElement.style.display="block";
+            }
+            else {
+                targetElement.style.display="none";
+            }
+        },
+        moveItem: function(olist, key, id) {
+            //for creating a new list
```

Comment:
Just a note relating the format: We always keep a space between // and the comment
```suggestion
            // for creating a new list
```